Source/Kysect.Shreks.WebApi/Program.cs

```diff
@@ -46,7 +47,13 @@ void InitServiceCollection(WebApplicationBuilder webApplicationBuilder)
     {
         webApplicationBuilder.Services.TryAddSingleton(testEnvironmentConfiguration);
     }
-    webApplicationBuilder.Services.AddControllers(x => x.Filters.Add<AuthenticationFilter>()).AddNewtonsoftJson();
+
+    webApplicationBuilder.Services
+        .AddControllers(x => x.Filters.Add<AuthenticationFilter>())
+        .AddNewtonsoftJson()
+        .AddApplicationPart(typeof(IControllerProjectMarker).Assembly)
```

Comment:
```suggestion
        .AddApplicationPart(typeof(IControllersProjectMarker).Assembly)
```

Если выше да, то тут тоже поменять надо будет.